src/index.html

```diff
@@ -1,13 +1,49 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="
+      width=device-width,
+      user-scalable=no,
+      initial-scale=1.0,
+      maximum-scale=1.0,
+      minimum-scale=1.0
+   ">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img
+        class="card__pic"
+        src="/images/imac.jpeg"
+        alt="iMac"
+      >
+      <p class="card__name">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</p>
+      <span class="card__code">Код товара: 195434</span>
+      <div class="raiting">
```

Comment:
Don't make BEM structure too complicated. You don't need to create separate blocks for anything other than card itself and stars.